Resource.rdef

```diff
@@ -0,0 +1,1143 @@
+		$"81C3EEFF02598FFB02598F897774770077747700777477007774770077747700"
+		$"7774770001588E5002598FFFAED2EAF9539BCAFFB9E0FBFFBCE2FDFFBCE2FDFF"
+		$"B1DCF9FF226F9FFFB2CFE1FA1A6999FF01588E50777477007774770077747700"
+		$"01588E4502598FFFD1ECFEFFA5DAFEFF6AAFDCFF6BA7CEFF92C2E2FF92C2E2FF"
+		$"68A3C9FF6CB0DCFFA5DAFEFFD8EDFBFF02598FFF01588E4F7774770077747700"
+		$"02598F98AFC5D3FFA0D8FEFF96D4FEFF96D4FEFF86C7F2FF75B8E4FF75B8E4FF"
+		$"7CBDE9FF78A9ECFF0000A4FF80ACECFFAEC1CCFF02598FA97774770077747700"
+		$"02598FFFC6E5FAFFB3E0FEFF96D4FEFF96D4FEFF96D4FEFF96D4FEFF96D4FEFF"
+		$"96D4FEFF0000A4FF0000A4FF0000A4FFB3D1E6FF02598FFF7774770077747700"
+		$"02598FFF94CEF5FFA9DAFCFFCDEAFDFFC2E5FEFFA7DBFEFF96D4FEFF9DD7FEFF"
+		$"6A8BDDFF0000A4FF0000A4FF0000A4FF375FB5FF02598FFF7774770077747700"
+		$"02598FFF90CDF5FF2EA7F9FFA8D9FCFFC7E7FEFFC3E6FEFFC0E3FAFFB4DAF5FF"
+		$"0000A4FF0000A4FFFFFFFFFF0000A4FF0000A4FF02598FFF7774770077747700"
+		$"02598FFF89C3EAFF41B0FAFFA2D6FCFFA3D7FDFF57BAFDFF58B4F1FF336CCEFF"
+		$"0000A4FFBBBBE5FFC0C0C0FFAAAADFFF0000A4FF013896FF7774770077747700"
+		$"535755FFA9B8C0FF4FB3F7FF7FC9FCFF7CC9FDFF56BAFDFF46ACF0FF0000A4FF"
+		$"0000A4FFFFFFFFFF535353FFFFFFFFFF0000A4FF0000A4FF7774770077747700"
+		$"535755FFA6AEB0FF909593FF88B4CFFF69BFF8FF5BB8F7FF386DCCFF0000A4FF"
+		$"5B5BC4FFFFFFFFFF292929FFFFFFFFFF5B5BC4FF0000A4FF0000A45C00000008"
+		$"535755FFA5ADAEFF979B9AFFA4A8A7FFC2C5C4FFB8BBBAFF0000A4FF0000A4FF"
+		$"FFFFFFFFFFFFFFFFCCCCCCFFFFFFFFFFFFFFFFFF0000A4FF0000A4FF0000001B"
+		$"535755FFADB5B6FCAAAEACFF999D9CFFAEB1B0FF7375B0FF0000A4FF7B7BCFFF"
+		$"FFFFFFFFFFFFFFFF292929FFFFFFFFFFFFFFFFFF5B5BC4FF0000A4FF0000876F"
+		$"2D32322E535755FFA9ACADFEC0C2C2FFB7BAB9FF0000A4FF0000A4FFFFFFFFFF"
+		$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF0000A4FF0000A4FF"
+		$"00000001262E3136535755FF636765FFAEB6BAFD0000A4FF0000A4FF0000A4FF"
+		$"0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF"
+		$"77747700777477000000000E393C3B613A494EB5353771FF0000A4FF0000A4FF"
+		$"0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF0000A4FF0000A45C"
+	}
+};
+
+resource large_icon array {
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8C8666668CFFFFFFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF666666666666668C8CFFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF86606018608D86868C8CFFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8C66118DD587878D8C8C8C8CFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8DB48786AEAE87AD8D8DAD8CFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8C8DD58DB487B48DB38DADB38CFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8C8DAE8DB4B4D5ADB38DB3B48CFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8D8DB4B4ADB4B4B3B38DB4B48CFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFAD8DB4D5B4B4B3B48DADB4B48DFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFAD8D8DADB3B4B487B4B4B48DFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8D8DB3B3B4B4B48D8DB4D4B3FFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF8D8DB3B4D4B48D87B4D4D48DFFFFFF"
+	$"FFFFFFFFFF
```

Comment:
Obvious duplicate of line 1139.
Please move app_signature and file_types to the top of the file.